### PR TITLE
subfinder: 2018-07-15 -> 2.2.4

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Motivation for this change
https://github.com/projectdiscovery/subfinder/releases/tag/v2.2.2
https://github.com/projectdiscovery/subfinder/releases/tag/v2.2.4

###### Things done

- Changed `owner` from `subfinder` to `projectdiscovery`
- Updated `deps.nix`

---

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
```
/nix/store/h6ab2zxcilm3chd8dhz0x7nd3yxqfs6j-subfinder-git-2018-07-15-bin	  36.7M
/nix/store/820pg6ynzc7rqb9sqyd2annh16vwh21x-subfinder-2.2.2-bin	  39.5M
```
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @fpletz

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/75234)

### CODE REVIEW

File: pkgs/tools/networking/subfinder/default.nix [hunk 1]
```diff
@@ -22,8 +22,8 @@ buildGoPackage {
       subdomains for websites. Designed as a passive framework to be
       useful for bug bounties and safe for penetration testing.
     '';
-    homepage = https://github.com/subfinder/subfinder;
+    homepage = "https://github.com/projectdiscovery/subfinder";
```
Comment (ehmry) [inline pkgs/tools/networking/subfinder/default.nix]: This could also be taken from `src.meta.homepage`.

**Review by ghost — APPROVED** [review]

File: pkgs/tools/networking/subfinder/default.nix (hunk 1 above)
Comment (filalex77) [reply pkgs/tools/networking/subfinder/default.nix]: I believe this is [taken by default](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchgithub/default.nix#L33). Should this be spelled out explicitly?

Comment (ehmry) [reply pkgs/tools/networking/subfinder/default.nix]: Perhaps not, I did not know it was taken by `mkDerivation`.

Comment (filalex77) [reply pkgs/tools/networking/subfinder/default.nix]: I think that it serves well for documentation purposes, and is similar to most other derivations.

**Pull request merged**